### PR TITLE
Feature - vcf-files

### PR DESCRIPTION
**Pull request by viklund**
Changed to using vcf-files instead of bed-files both internally and for result files.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -79,24 +79,16 @@ process manta {
     module 'bioinfo-tools'
     module "$params.modules.manta"
 
+    errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
+    time { params.long_job * 2**task.attempt }
```
Comment (andkaha) [inline main.nf]: What's the effect of this? First retry will get double time, second quadruple, etc.?

File: main.nf [hunk 2]
```diff
@@ -201,29 +193,30 @@ process mask_beds {
     module "$params.modules.bedtools"
 
     """
-    BNAME=\$( echo $bedfile | cut -d. -f1 )
-    MASK_FILE=\${BNAME}_masked.bed
-    cat $bedfile \
-        | bedtools intersect -v -a stdin -b $mask1 -f 0.25 \
-        | bedtools intersect -v -a stdin -b $mask2 -f 0.25 > \$MASK_FILE
+    BNAME=\$( echo $svfile | cut -d. -f1 )
```
Comment (andkaha) [inline main.nf:196]: Or `\${svfile%%.*}` if `svfile` is a shell variable, which I think it ought to be.

File: main.nf [hunk 3]
```diff
@@ -294,6 +282,12 @@ process variant_effect_predictor {
               exit 1;;
     esac
 
+    ## If the input file is empty, just copy
+    if [ \$( wc -l "\$infile" | awk '{print \$1}' ) -eq 0 ]; then
```
Comment (andkaha) [inline main.nf:286]: `if [[ -f "\$infile" && ! -s "\$infile" ]]; then`

**Review by andkaha — COMMENTED** [review]
Looks generally ok.

File: main.nf (hunk 1 above)
Comment (viklund) [reply main.nf]: Yes

File: main.nf (hunk 2 above)
Comment (viklund) [reply main.nf:196]: Will fix

File: main.nf (hunk 3 above)
Comment (viklund) [reply main.nf:286]: Will correct

**Pull request merged**